### DISCUSSION TITLE
alist只有onedrive支持目录容量显示

### DISCUSSION THREAD

**Discussion by fzlychee2019** [AlistGo/alist#1140] [General]
其他云盘啥时也能支持目录容量显示呢？有了目录容量显示，会方便太多太多了，不然很难判断某个目录文件是否都上传成功了，有了目录容量显示，就能非常直观看出每个目录容量了。

**Comment by xhofe** [comment]
不支持。

**Reply by pdj9na** [reply]
我也有这样的需求，这是个很关键的功能，不然就没法和 rclone 比了，只要 rclone 支持的，都可以不用这个，阿里云盘 的挂载还不如使用 clouddrive

**Comment by anwen-anyi** [comment]
去问那些网盘为啥不支持哈哈   他们不支持也木的办法 onedrive显示是因为onedrive本来就显示~ 别的网盘不显示~~

**Reply by ma3252788** [reply]
> 网盘基本都支持显示容量的

大佬，请问天翼云网盘怎么显示容量呀？找了好久了

**Reply by ma3252788** [reply]
> > > 网盘基本都支持显示容量的
> > 
> > 
> > 大佬，请问天翼云网盘怎么显示容量呀？找了好久了
> 
> 页面左下角不就有吗

我的意思是像OneDrive那种，显示每个文件夹的容量。天翼云只支持查看剩余总容量。
